django_fsm_freeze/models.py

```diff
@@ -31,10 +34,14 @@ def bypass_fsm_freeze(objs: Union[object, Iterable]):
         raise FreezeConfigurationError(errors)
 
     try:
+        if bypass_globally is True:
+            setattr(FreezableFSMModelMixin, '_DISABLED_FSM_FREEZE', True)
```

Comment:
it could be
```suggestion
            FreezableFSMModelMixin._DISABLED_FSM_FREEZE = True
```

Why using `setattr` ? without `setattr`, the type checker will catches typo.